hydra-node/src/Hydra/API/ServerOutput.hs

```diff
@@ -6,14 +6,13 @@ import Hydra.API.ClientInput (ClientInput (..))
 import Hydra.Chain (PostChainTx, PostTxError)
 import Hydra.Crypto (MultiSignature)
 import Hydra.Ledger (IsTx, UTxOType, ValidationError)
-import Hydra.Network (Host)
 import Hydra.Party (Party)
 import Hydra.Prelude
 import Hydra.Snapshot (Snapshot, SnapshotNumber)
 
 data ServerOutput tx
-  = PeerConnected {peer :: Host}
-  | PeerDisconnected {peer :: Host}
+  = PeerConnected {peer :: Natural}
```

Comment:
Should we introduce a type alias for NodeId so we know what are we referring to?